codebuild/spec/buildspec_omnibus.yml

```diff
@@ -186,6 +186,14 @@ batch:
           GCC_VERSION: '6'
           TESTS: crt
 
+    - identifier: s2nUnitInstallSharedAndStatic
+      buildspec: codebuild/spec/buildspec_ubuntu.yml
+      env:
+        compute-type: BUILD_GENERAL1_SMALL
+        privileged-mode: true
+        variables:
+          TESTS: sharedandstatic
+
```

Comment:
Add a `S2N_LIBCRYPTO=openssl-1.1.1` environment variable to make it explicit which libcrypto we want.